crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -319,7 +312,25 @@ impl<'a> VM<'a> {
             .try_into()
             .map_err(|_err| ExceptionalHalt::VeryLargeNumber)?;
 
-        if Self::is_valid_jump_addr(call_frame, jump_address_usize) {
```

Comment:
Why not keep the helper? The code is rather complex that makes it more self-explanatory in call-context.

Reply:
It was only used there. Fixed, now it's a helper again.